v-spec.adoc

```diff
@@ -4571,7 +4571,7 @@ If XLEN > SEW, _OFFSET_ is _not_ truncated to SEW bits.
                VLMAX <= i+OFFSET           src[i] = 0
 
   vslidedown behavior for destination element i in slide
-                   0 <  i < vstart         Unchanged
+                   0 <= i < vstart         Unchanged
```

Comment:
Like above, I think the upper bound is meant to be understood as `min(vl, vstart)`. If we add an explicit assumption that `vstart < vl`, we can avoid making any changes.